playwright.config.ts

```diff
@@ -14,13 +14,13 @@ import { GitHubActionOptions } from '@estruyf/github-actions-reporter';
  */
 export default defineConfig({
   testDir: './src/__tests__',
-  snapshotPathTemplate: '{testDir}/visual/{testName}/{arg}{ext}',
+  snapshotPathTemplate: '{testDir}/{testFileDir}/__screenshoots__/{testName}/{arg}{ext}',
   expect: {
     // Maximum time expect() should wait for the condition to be met.
     timeout: 5000,
 
     toHaveScreenshot: {
-      maxDiffPixelRatio: 0.01,
+      maxDiffPixelRatio: 0.02,
```

Comment:
If you don't have any next task, maybe you could try to change the yarn action to run those tests in docker (both run test and update snapshots, so we could make this ration smaller)

Reply:
I thought about creating a task for that and try to solve this issue. 
https://bitcoinassociation.atlassian.net/browse/SPV-1105